src/components/_app/Web3ConnectionManager/components/WalletSelectorModal/components/GoogleLoginButton/components/CopyCWaaSBackupData.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Icon,
+  IconButton,
+  Text,
+  Tooltip,
+  VStack,
+  useClipboard,
+} from "@chakra-ui/react"
+import { usePostHogContext } from "components/_app/PostHogProvider"
+import Button from "components/common/Button"
+import useSubmit from "hooks/useSubmit"
+import useToast from "hooks/useToast"
+import { Copy } from "phosphor-react"
+import { useEffect } from "react"
+import useDriveOAuth from "../hooks/useDriveOAuth"
+import { getDriveFileAppProperties, listWalletsOnDrive } from "../utils/googleDrive"
+
+const CopyCWaaSBackupData = () => {
+  const { captureEvent } = usePostHogContext()
+  const driveOAuth = useDriveOAuth()
+  const toast = useToast()
+  const {
+    onCopy,
+    setValue: setBackup,
+    value: backup,
+    hasCopied,
+  } = useClipboard("", 5000)
+
+  useEffect(() => {
+    if (!hasCopied) return
+
+    toast({
+      status: "success",
+      title: "Copied!",
+      description: "Backup data successfully copied to the clipboard",
+    })
+  }, [hasCopied])
+
+  // This toast is needed, because we can't copy to clipboard immediately after the submit, due to browser limitations
+  useEffect(() => {
+    toast({
+      status: "info",
+      title: "Backup downloaded",
+      description: (
```

Comment:
We could use useToastWithButton here

Reply:
Thanks! Changed in https://github.com/guildxyz/guild.xyz/pull/1015/commits/37b5ea9f9f48e8d3ea4a72304419a954d28126c7 and https://github.com/guildxyz/guild.xyz/pull/1015/commits/12e3bf112f96b0a2c0348c4ce42e4092414d464b